### ISSUE TITLE
In dark mode, cannot see highlights

### ISSUE THREAD

**Issue by lakdafari** [elias-sundqvist/obsidian-annotator#298]
First off, amazing plugin! Thank you!

For some reason, I cannot see the highlights in yellow when I switch to dark mode. They are there - and when I hover over them the pointer changes - but the actual highlight colour does not overlay on the PDF. In light mode it works as expected.

Please let me know if I can customize a setting to make this work for me. Thank you!

**Comment by LightNeko** [comment]
I also noticed a similar problem, for meit  is not that I can't see the highlights, but it is barely noticeable making it very difficult to read.

Here's an image that shows how it appears on dark mode to me:
![annotator-issue-1](https://user-images.githubusercontent.com/84442556/219955779-b29baff2-cc82-488a-9f23-5ad1705c3adb.png)

**Comment by NoughtKin** [comment]
> I also noticed a similar problem, for me, it is not that I can't see the highlights, but it is barely noticeable making it very difficult to read.
> 
> Here's an image that shows how it appears on dark mode to me: ![annotator-issue-1](https://user-images.githubusercontent.com/84442556/219955779-b29baff2-cc82-488a-9f23-5ad1705c3adb.png)

Yeah I also have a hard time looking for highlights when in dark mode

**Comment by NoughtKin** [comment]
Yeah I also have a hard time looking for highlights when in dark mode



> I also noticed a similar problem, for me, it is not that I can't see the highlights, but it is barely noticeable making it very difficult to read.
> 
> Here's an image that shows how it appears on dark mode to me: ![annotator-issue-1](https://user-images.githubusercontent.com/84442556/219955779-b29baff2-cc82-488a-9f23-5ad1705c3adb.png)

**Comment by SamCao920** [comment]
I've had the same issue. The highlights are very difficult to see.

<img width="887" height="67" alt="Image" src="https://github.com/user-attachments/assets/203133d9-86af-4b43-87e4-a73c09a1c2d3" />

**Comment by shravya-katapally** [comment]
Yep, same issue here

<img width="1322" height="693" alt="Image" src="https://github.com/user-attachments/assets/1883dad0-d1ae-46cb-89a3-4cc63e2bbc46" />
